src/components/dashboard/utils/sendReceiveFlow.js

```diff
@@ -1,17 +1,47 @@
+import React, { useContext } from 'react'
 import { t } from '@lingui/macro'
+import Config from '../../../config/config'
+import AppNavBar from '../../appNavigation/NavBar'
+import { TokenContext } from '../../../lib/wallet/GoodWalletProvider'
+import { NetworkName } from '../../appNavigation/TabsView'
+
+const NavBar = ({ title, goBack }) => {
+  const { token } = useContext(TokenContext)
+
+  return (
+    <AppNavBar goBack={goBack} title={`${title} ${token}`}>
+      <NetworkName icon={false} />
+    </AppNavBar>
+  )
+}
 
 export const navigationOptions = ({ navigation, screenProps }) => {
+  const { routeName } = navigation.state
   const action = navigation.getParam(PARAM_ACTION)
+
   const isBridge = navigation.getParam('isBridge')
+  const isReceive = action === ACTION_RECEIVE || ['Receive', 'ReceiveToAddress'].includes(routeName)
 
-  return {
-    title: action === ACTION_RECEIVE ? RECEIVE_TITLE : isBridge ? BRIDGE_TITLE : SEND_TITLE,
+  let options = {
+    title: isReceive ? RECEIVE_TITLE : isBridge ? BRIDGE_TITLE : SEND_TITLE,
     isBridge,
   }
+
+  if (Config.isDeltaApp && !isBridge) {
+    options.title = isReceive ? RECEIVE_NATIVE_TITLE : SEND_NATIVE_TITLE
+
+    if (['Amount', 'Receive'].includes(routeName)) {
+      options.navigationBar = NavBar
+    }
+  }
+
+  return options
 }
 
 export const RECEIVE_TITLE = t`Receive G$`
```

Comment:
as was requested - all UI changes are under delta flag. old tiles are still used in the default GD app
vvv
```
 if (Config.isDeltaApp && !isBridge) {
    options.title = isReceive ? RECEIVE_NATIVE_TITLE : SEND_NATIVE_TITLE
```

technically it's possible, but let it be a separate ticket (e.g. "ETH support - enhancement") because eToro wants this PR to be merged ASAP as they're already awaited too long. WDYT ? Shall I open a ticket ?